src/components/icons.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { SVGProps } from 'react';
+
+type Icon = SVGProps<SVGSVGElement>;
+
+export enum Color {
+  Active = '#0ba8e0',
+  Inactive = '#7a8287',
+}
+
+export const ThumbsUpIcon = (props: Icon): JSX.Element => (
+  <svg
+    {...props}
+    className="raf-reaction-icon__image"
+    data-icon="thumbs-up"
+    role="img"
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 512 512"
```

Comment:
same for other Icons to allow overriding the default  attributes
```suggestion
    className="raf-reaction-icon__image"
    data-icon="thumbs-up"
    role="img"
    xmlns="http://www.w3.org/2000/svg"
    viewBox="0 0 512 512"
     {...props}
```